src/cfml/system/endpoints/ForgeBox.cfc

```diff
@@ -181,6 +181,49 @@ component accessors="true" implements="IEndpointInteractive" singleton {
 		}
 	}	
 
+	/*
+	* Figures out what version of a package would be installed with a given semver range without actually going through the installation.
+	* @slug Slug of package
+	* @version Version range to satisfy
+	* @entryData Optional struct of entryData which skips the ForgeBox call.
+	*/
+	function findSatisfyingVersion( required string slug, required string version, struct entryData ) {
+		
+			// Use passed in entrydata, or go get it from ForgeBox.
+			var entryData = arguments.entryData ?: forgebox.getEntry( arguments.slug );
+			
+			entryData.versions.sort( function( a, b ) { return semanticVersion.compare( b.version, a.version ) } );
```

Comment:
Dunno, you can if you want but you'll need to implement the full semver comparison on the server side to do so.